### PR TITLE
[FOGL-5027]: Fixed issue of update schedule due to time field

### PR DESCRIPTION
**Pull request by monika-Dianomic**

### CODE REVIEW

File: src/app/components/core/scheduler/update-schedule/update-schedule.component.ts [hunk 1]
```diff
@@ -144,6 +145,9 @@ export class UpdateScheduleComponent implements OnInit, OnChanges {
             time: time.time,
             enabled: data['enabled']
           });
+          if (this.form.get('time').value === '00:00:00') {
+            this.form.controls['time'].setErrors({ invalid: true });
```
Comment (praveen-garg) [inline src/app/components/core/scheduler/update-schedule/update-schedule.component.ts:149]: why is this  invalid, the midnight time?

File: src/app/components/core/scheduler/update-schedule/update-schedule.component.ts [hunk 2]
```diff
@@ -190,6 +194,9 @@ export class UpdateScheduleComponent implements OnInit, OnChanges {
     let dayIndex;
     if (this.selectedScheduleTypeIndex === 2) {   // If Type is TIMED == 2
       time = Utils.convertTimeToSec(this.form.get('time').value);
+      if (time === 0) {
```
Comment (praveen-garg) [inline src/app/components/core/scheduler/update-schedule/update-schedule.component.ts:197]: add comment

File: src/app/components/core/scheduler/update-schedule/update-schedule.component.html [hunk 3]
```diff
@@ -130,9 +130,9 @@
             <div class="field">
               <label class="label is-small">Time</label>
               <p class="control is-fullwidth">
-                <input class="input is-small" type='text' placeholder="hh:mm:ss" mask="00:00:00" formControlName="time">
+                <input class="input is-small" type='text' placeholder="HH:mm:ss" mask="00:00:00" formControlName="time" (input)="matchTimeTypeField($event, 'time')">
               </p>
-              <small *ngIf="!form.controls.time.valid" class="help is-danger level-left">&nbsp; Time must be in range
+              <small *ngIf="!form.controls.time.valid || form.controls.time.value === '00:00:00'" class="help is-danger level-left">&nbsp; Time must be in range
                 00:00:01 - 23:59:59</small>
```
Comment (praveen-garg) [inline src/app/components/core/scheduler/update-schedule/update-schedule.component.html:136]: should be 00:00:00 to 23:59:59

File: src/app/components/core/scheduler/update-schedule/update-schedule.component.ts (hunk 1 above)
Comment (monika-Dianomic) [reply src/app/components/core/scheduler/update-schedule/update-schedule.component.ts:149]: Yes, it is invalid. Getting error from backend if time is 0.

**Review by praveen-garg — APPROVED** [review]
lgtm!

**Review by mshariq-nerd — APPROVED** [review]

**Pull request merged**